core/services/keystore/keys/ocr2key/offchain_keyring.go

```diff
@@ -62,13 +66,18 @@ func (ok *OffchainKeyring) OffchainPublicKey() ocrtypes.OffchainPublicKey {
 }
 
 func (ok *OffchainKeyring) ConfigEncryptionPublicKey() ocrtypes.ConfigEncryptionPublicKey {
+	cpk, _ := ok.configEncryptionPublicKey()
+	return cpk
```

Comment:
@connorwstein Since we can't return an error here, what do you think about validating from `unmarshal` and the constructor instead, so that we can ignore the error here?

Reply:
makes sense to me, good defensive move